experiments/run_recourse_experiment.py

```diff
@@ -343,12 +355,14 @@ def run_dice(
         random_seed=poi_seed,
     )
 
+    start_time = time.time()
+
     # Generate the paths.
     paths = iterator.iterate_k_recourse_paths(
         poi=poi, max_iterations=max_iterations
     )
 
-    return paths
+    return paths, time.time() - start_time
```

Comment:
For readability's sake, I'd suggest assigning the elapsed time to a variable (e.g. something descriptive of your choosing, like `dice_elapsed_time` or `elapsed_recourse_clock_time`, etc.) and returning the variable. Might be useful to note the units in the name.